system-parachains/encointer/src/lib.rs

```diff
@@ -624,16 +570,74 @@ impl pallet_membership::Config<pallet_membership::Instance1> for Runtime {
 	type WeightInfo = weights::pallet_membership::WeightInfo<Runtime>;
 }
 
+/// A `HandleCredit` implementation that naively transfers the fees to the block author.
+/// Will drop and burn the assets in case the transfer fails.
+pub struct AssetsToBlockAuthor<R>(PhantomData<R>);
```

Comment:
In other system paras we send only 50% to the author and 50% to a pot to share with other collators. If you send 100% to the author, then the author can attempt to increase the fee rate by making transactions up to the block limit at zero cost (since they pocket the fees). Sharing amongst others would make them at least collude to drive up the fee rate.

Impl here: https://github.com/paritytech/polkadot-sdk/blob/4e10d3b0a6ec2eccf58c471e7739948c1a867acf/cumulus/parachains/common/src/impls.rs#L62

The pot actually takes care of the 50/50 part.

Reply:
I think I addressed your valid concern by burning 50% of fees if they are paid in community currency. In the future, we'll direct a fraction to community treasuries instead of burning. Left a comment in the respective code